cmd/util/util.go

```diff
@@ -165,3 +167,89 @@ func PromptUserForAccept(r io.Reader) bool {
 		return false
 	}
 }
+
+func MaybePrintKubectlDownloadMsg() {
+	if !viper.GetBool(config.WantKubectlDownloadMsg) {
+		return
+	}
+
+	// darwin, linux and windows are only available k8s url options
+	//   all possible runtime.GOOS options: "android darwin dragonfly freebsd linux nacl netbsd openbsd plan9 solaris windows "
```

Comment:
I believe arm and 386 are architecture types and are mentioned in the next set of comments (line 179-180).

The reason I did not inject runtime.GOOS was because, for example someone on solaris/android should be able to use a linux binary and thusly they should get the linux URL for kubectl.  It is minor though,  unclear which is better now.  I am leaving as is for now.  

Regarding the windows instructions I can change them but it is not clear to me that the alternative is better.